webapp/channels/src/components/forward_post_modal/forward_post_modal.tsx

```diff
@@ -228,7 +228,7 @@ const ForwardPostModal = ({onExited, post, actions}: Props) => {
 
     const postPreviewFooterMessage = formatMessage({
         id: 'forward_post_modal.preview.footer_message',
-        defaultMessage: 'Originally posted in ~{channelName}',
+        defaultMessage: 'Originally posted in ~{channel}',
```

Comment:
Seems safer to change the value passed instead of the string. Keep in mind that changing the string means it has to be translated all over again in all languages. And the value make sense (`channelName` and not `channel`).